main.tf

```diff
@@ -224,6 +224,28 @@ resource "aws_ec2_transit_gateway_vpc_attachment" "default" {
   transit_gateway_id                              = var.transit_gateway_id
   vpc_id                                          = aws_vpc.default.id
   tags                                            = { Name = var.name }
+
+  lifecycle {
+    ignore_changes = [
+      transit_gateway_default_route_table_association, transit_gateway_default_route_table_propagation
+    ]
+  }
+}
+
+resource "aws_ec2_transit_gateway_vpc_attachment_accepter" "default" {
```

Comment:
https://github.com/schubergphilis/terraform-aws-mcaf-transit-gateway/blob/main/main.tf#L214C11-L214C58 mimic

Reply:
set defaults to false explicitely and updated tagging